test/ui/ui-main.swapping.test.ts

```diff
@@ -0,0 +1,149 @@
+/* eslint-disable no-console */
+import { Mangata } from "../../utils/frontend/pages/Mangata";
+import { Keyring } from "@polkadot/api";
+import { BN } from "@polkadot/util";
+import { WebDriver } from "selenium-webdriver";
+import { getApi, initApi } from "../../utils/api";
+import { Assets } from "../../utils/Assets";
+import { Sidebar } from "../../utils/frontend/pages/Sidebar";
+import { DriverBuilder } from "../../utils/frontend/utils/Driver";
+import {
+  addExtraLogs,
+  setupPolkadotExtension,
+  uiStringToBN,
+} from "../../utils/frontend/utils/Helper";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { Sudo } from "../../utils/sudo";
+import { AssetWallet, User } from "../../utils/User";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import { Swap } from "../../utils/frontend/pages/Swap";
+import { MGA_ASSET_ID, MGA_ASSET_NAME } from "../../utils/Constants";
+import {
+  ModalType,
+  NotificationModal,
+} from "../../utils/frontend/pages/NotificationModal";
+import { Polkadot } from "../../utils/frontend/pages/Polkadot";
+import { waitNewBlock } from "../../utils/eventListeners";
+import { createAssetIfMissing, createPoolIfMissing } from "../../utils/tx";
+import { SudoUser } from "../../utils/Framework/User/SudoUser";
+import { Node } from "../../utils/Framework/Node/Node";
+import { connectPolkadotWallet } from "../../utils/frontend/utils/Handlers";
+
+require("dotenv").config();
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+
+jest.setTimeout(1500000);
+let driver: WebDriver;
+process.env.NODE_ENV = "test";
```

Comment:
plz, do not manipulate env. variables inside tests :O

Reply:
yep, removed that - it was a leftover I missed, thx!